app/javascript/components/item-view.vue

```diff
@@ -3,9 +3,12 @@ import { ref } from 'vue';
 import { useNotification } from '@kyvg/vue3-notification';
 import purchasesApi from '../api/purchases';
 import type { Item } from '../api/items';
+import type { Review } from '../api/reviews';
```

Comment:
🚫 **[eslint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'Review' is defined but never used  @typescript-eslint/no-unused-vars